test/EIP20CoGateway/isRedeemApproved.js

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const chai = require('chai');
+const Web3 = require('web3');
+const sinon = require('sinon');
+
+const assert = chai.assert;
+const EIP20CoGateway = require('../../src/ContractInteract/EIP20CoGateway');
+const EIP20Token = require('../../src/ContractInteract/EIP20Token');
+const SpyAssert = require('../../test_utils/SpyAssert');
+const AssertAsync = require('../../test_utils/AssertAsync');
+
```

Comment:
up until here, same comments as above.

Reply:
Yea, we can move to test directory.   I think it can be done for all the test files after everything is merged.
 I can create a cleanup ticket and can be quickly done at the end. Maybe after release 🤔